deploy/examples/import-external-cluster.sh

```diff
@@ -76,87 +76,131 @@ function importClusterID() {
 }
 
 function importSecret() {
-  kubectl -n "$NAMESPACE" \
-    create \
-    secret \
-    generic \
-    --type="kubernetes.io/rook" \
-    "$MON_SECRET_NAME" \
-    --from-literal="$MON_SECRET_CLUSTER_NAME_KEYNAME"="$ROOK_EXTERNAL_CLUSTER_NAME" \
-    --from-literal="$MON_SECRET_FSID_KEYNAME"="$ROOK_EXTERNAL_FSID" \
-    --from-literal="$MON_SECRET_ADMIN_KEYRING_KEYNAME"="$ROOK_EXTERNAL_ADMIN_SECRET" \
-    --from-literal="$MON_SECRET_MON_KEYRING_KEYNAME"="$ROOK_EXTERNAL_MONITOR_SECRET" \
-    --from-literal="$MON_SECRET_CEPH_USERNAME_KEYNAME"="$ROOK_EXTERNAL_USERNAME" \
-    --from-literal="$MON_SECRET_CEPH_SECRET_KEYNAME"="$ROOK_EXTERNAL_USER_SECRET"
+  secret=$(kubectl -n "$NAMESPACE" get secret "$MON_SECRET_NAME" 2>a.txt)
+  if [ -s a.txt ]; then
```

Comment:
Actually instead of piping stderr to a file, how about checking the error code? Something like this:
```suggestion
  if [ $? == 0 ]; then
```
Then there is no file to worry about

Reply:
Updated thanks